data/json/itemgroups/Weapons_Mods_Ammo/gunmod.json

```diff
@@ -21,7 +21,9 @@
       [ "cheek_pad", 25 ],
       [ "rail_laser_sight", 40 ],
       [ "red_dot_sight", 70 ],
-      [ "choke", 50 ]
+      [ "choke", 50 ],
+      [ "big_brass_catcher", 20 ],
```

Comment:
I don't think brass catchers should be in gunstores, they're too handmade. Yes, there's ones you can buy at the store, but those are nylon and different than the ones currently in game. You can leave this in if you'd like, I can make the store version in another PR.

Reply:
I put it in the same pool as the regular brass catcher with 1/4th the spawn rate because the only other pools I saw were police and military specific stuff, and I'm not sure they fall under that. I did look for more curated spawn pools but I couldn't find any. You're welcome to move stuff around if it feels necessary, though.